Example/Info.plist

```diff
@@ -25,7 +25,7 @@
 	<key>NSLocationWhenInUseUsageDescription</key>
 	<string>Demo app helps you find more information near you</string>
 	<key>RASProjectId</key>
-	<string>$(RAS_APPLICATION_IDENTIFIER)</string>
+	<string>$(RAS_PROJECT_ID)</string>
```

Comment:
I think you can keep this as `RAS_APPLICATION_IDENTIFIER` for now. It also needs to be updated in the `configure-secrets.sh` script and then updated in all of the CI settings and on our local machines. So it will be kind of a pain to change 😣  I think we can handle this later.

Reply:
Ok 👍